xonsh/environ.py

```diff
@@ -191,6 +194,7 @@ def xonshconfig(env):
     'PUSHD_SILENT': False,
     'RAISE_SUBPROC_ERROR': False,
     'RIGHT_PROMPT': '',
+    'DYNAMIC_CWD_WIDTH': '20',
```

Comment:
Do you mean that if defaulted to 'infinite' it could just replace the standard {cwd}?